rfcs/interop_2022.md

```diff
@@ -0,0 +1,133 @@
+
+### New Viewport Units
+
+Proposal: https://github.com/web-platform-tests/interop-2022/issues/4  
+Tests: `interop-2022-viewport` (TODO)
+
+See also the [Viewport Investigation project](https://github.com/web-platform-tests/interop-2022/issues/41) which is part of the Interop 2022 effort, but not part of the metric.
+
+### Web Compat
+
+Proposal: https://github.com/web-platform-tests/interop-2022/issues/9  
+Tests: [`interop-2022-webcompat`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2022-webcompat)
+
+### Interop 2021
+
+The 5 focus areas from Interop 2021 are carried forward and included in the metric. See https://github.com/web-platform-tests/interop-2022/issues/2 for discussion.
+
+- [`interop-2021-aspect-ratio`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-aspect-ratio)
+- [`interop-2021-flexbox`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-flexbox)
+- [`interop-2021-grid`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-grid)
+- [`interop-2021-position-sticky`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-position-sticky)
+- [`interop-2021-transforms`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-transforms)
+
+[Query for `interop-2021-*`](https://wpt.fyi/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&q=label%3Ainterop-2021-aspect-ratio%20or%20label%3Ainterop-2021-flexbox%20or%20label%3Ainterop-2021-grid%20or%20label%3Ainterop-2021-position-sticky%20or%20label%3Ainterop-2021-transforms)
+
+### Metrics
+
+Scores (between 0% and 100%) will be computed from test suite pass rates for each focus area and the effort as a whole. In total there are 15 focus areas, of which 5 are carried forward from Interop 2021. All areas are given equal weight, meaning that the new areas make up 2/3 of the score and the old ones 1/3 of the score.
+
+Scoring in detail:
+
+- Every test is scored between 0 and 1. For tests with subtests, the score is the number of passes divided by the number of subtests. This avoids tests with many subtests being given much larger weight than for example reftests.
+- Every area is scored between 0 and 1, adding up the scores and dividing by the number of tests.
+- The overall score is the sum of area scores divided by the number of areas (15).
+
+Scores are presented as percentages.
+
+TODO: decide on floating point vs. integer vs. rational numbers
```

Comment:
I don't think this kind of implementation detail needs to be in the RFC.

Reply:
Sure, I'll rewrite it all to a 0-100 score.